app/src/main/java/org/feature/fox/coffee_counter/data/local/database/dao/UserDao.kt

```diff
@@ -30,6 +36,9 @@ interface UserDao {
     @Delete
     suspend fun deletePurchase(purchase: Purchase)
 
+    @Delete
+    suspend fun deleteImage(image: Image)
+
```

Comment:
Pls add Db at the end of the method name to stay consistent and provide better Overview